docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+services:
+  prereqs:
+    build:
+      context: ./
+      dockerfile: ./net/grpc/gateway/docker/prereqs/Dockerfile
+    image: grpc-web:prereqs
+  echo-server:
+    build:
+      context: ./
+      dockerfile: ./net/grpc/gateway/docker/echo_server/Dockerfile
+    image: grpc-web:echo-server
+    ports:
+      - "9090:9090"
+  envoy:
+    build:
+      context: ./
+      dockerfile: ./net/grpc/gateway/docker/envoy/Dockerfile
+    image: grpc-web:envoy
+    network_mode: "host"
```

Comment:
Using `network="host"` is not supported on docker for Mac (see https://docs.docker.com/network/host/).

Using [`link`](https://docs.docker.com/compose/compose-file/#links) to connect services to envoy and then referencing service name in the envoy config works fine though.